flutter_vlc_player/ios/Classes/VlcViewController.swift

```diff
@@ -539,20 +602,24 @@ class VLCPlayerEventStreamHandler: NSObject, FlutterStreamHandler, VLCMediaPlaye
     }
 }
 
-enum DataSourceType: Int {
+enum DataSourceType: Int
+{
     case ASSET = 0
     case NETWORK = 1
     case FILE = 2
 }
 
-enum HWAccellerationType: Int {
+enum HWAccellerationType: Int
+{
```

Comment:
Should we rool-back those formatting changes to avoid tracking it in the diff? Any reason we want to re-format?